.github/workflows/ci.yml

```
@@ -0,0 +1,18 @@
+name: CI
+
+on:
+  pull_request:
+  push:
```

You probably want to restrict branches before merging this:

```suggestion
on:
  pull_request:
    branches:
      - master
  push:
    branches:
      - master
```